<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKxCombineKeyActionDialog</class>
 <widget class="QDialog" name="QKxCombineKeyActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>VT mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QRadioButton" name="radAnsiP">
          <property name="text">
           <string>+Ansi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAnsiN">
          <property name="text">
           <string>-Ansi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAnsiO">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QRadioButton" name="radNewLineP">
          <property name="text">
           <string>+NewLine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radNewLineN">
          <property name="text">
           <string>-NewLine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radNewLineO">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QRadioButton" name="radAppCuKeysP">
          <property name="text">
           <string>+AppCuKeys</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAppCuKeysN">
          <property name="text">
           <string>-AppCuKeys</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAppCuKeysO">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QRadioButton" name="radAppScreenP">
          <property name="text">
           <string>+AppScreen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAppScreenN">
          <property name="text">
           <string>-AppScreen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAppScreenO">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QRadioButton" name="radAppKeyPadP">
          <property name="text">
           <string>+AppKeyPad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAppKeyPadN">
          <property name="text">
           <string>-AppKeyPad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAppKeyPadO">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Shortcut</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QRadioButton" name="radShiftP">
          <property name="text">
           <string>+Shift</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radShiftN">
          <property name="text">
           <string>-Shift</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radShiftO">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QRadioButton" name="radCtrlP">
          <property name="text">
           <string>+Ctrl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radCtrlN">
          <property name="text">
           <string>-Ctrl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radCtrlO">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QRadioButton" name="radAltP">
          <property name="text">
           <string>+Alt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAltN">
          <property name="text">
           <string>-Alt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radAltO">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QRadioButton" name="radMetaP">
          <property name="text">
           <string>+Meta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radMetaN">
          <property name="text">
           <string>-Meta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radMetaO">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Key:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="key">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnQuickInput">
          <property name="text">
           <string>Quick input</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../private/skins/black/login.png</normaloff>../private/skins/black/login.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Combine key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ckey">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(224, 27, 36);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Action</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radString">
          <property name="text">
           <string>String to remote server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radOperation">
          <property name="text">
           <string>Operation on local terminal</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="opArea" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Operation:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="op">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="stringArea" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>String:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ansiText"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../private/skins/black/tips.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>[+]:Preceding a modename means the key/mode is pressed/active respectively.</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>[-]:Preceding a modename means the key/mode isn't pressed/active respectively.</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>[None]:ignore this key/mode.</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Executing a shortcut in the specified VT mode will trigger the corresponding action.</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
